referral-form-data-process/serverless.yml

```diff
@@ -23,6 +24,8 @@ functions:
     environment:
       CLIENTEMAIL: ${opt:CLIENTEMAIL}
       PRIVATEKEY: ${opt:PRIVATEKEY}
+      TEMPLATEDOCUMENTID: ${opt:TEMPLATEDOCUMENTID}
```

Comment:
Any chance we could put some underscores in these to make them more readable?

```
CLIENT_EMAIL
PRIVATE_KEY
TEMPLATE_DOCUMENT_ID
```

Reply:
Done in fa3a2fd